packages/app-store/routing-forms/pages/layout-handler/[...appPages].tsx

```diff
@@ -35,26 +35,3 @@ export default function LayoutHandler(props: { [key: string]: unknown }) {
     </FormProvider>
   );
 }
-
-LayoutHandler.getLayout = (page: React.ReactElement) => {
```

Comment:
not needed because routing-forms components no longer have getLayout (I removed all)